### PR TITLE
Audio device names is now shown correctly in the Audio Device combolist.

### PR DESCRIPTION
**Pull request by SoftAllan**
Ignored DA version of resource dll.

### CODE REVIEW

**Comment by pchote** [comment]
@RoosterDragon can you comment on this re #12167 and #12150?

**Comment by SoftAllan** [comment]
The screenshot below shows the encoding done with Default that uses the default system codepage. In this case the ISO-8859-1 codepage:
audiodevicebefore

It should be an UTF-8 encoding instead. This is after the change in the code:
audiodeviceafter

File: .gitignore [hunk 1]
```diff
@@ -80,3 +80,5 @@ StyleCopViolations.xml
 
 # Support directory
 /Support
+/da/Microsoft.Build.Utilities.v3.5.resources.dll
```
Comment (GraionDilach) [inline .gitignore]: What are these?

**Comment by RoosterDragon** [comment]
[Looks like UTF8 is the correct encoding](https://stackoverflow.com/questions/16068339/what-encoding-is-used-by-functions-such-as-glgetstring-and-glgetshaderinfolog). Don't know why I couldn't find that at the time, but I'm glad somebody noticed and made it better :)

:+1: modulo Graion's question.

**Comment by abcdefg30** [comment]
Removed the changes to `.gitignore`.

**Comment by abcdefg30** [comment]
This also got a second 👍 from @netnazgul (see #13837).

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/dac212638dffeea8c505a4bc99abd45967d0fc11...45a67813b669544b4d00e87008d217ac2a9b7041)

https://github.com/OpenRA/OpenRA/commit/521236398d7ed33cb8e63030f43d49d22634285b

**Pull request merged**